keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -431,10 +478,15 @@ We propose a single e2e test for the following scenario:
 - e2e tests
 - implement the `job_by_external_controller_total` metric
 - address all known inconsistencies between validation and the Job controller behavior,
-  in particular: [#123775](https://github.com/kubernetes/kubernetes/issues/123775)
+  in particular: [#123775](https://github.com/kubernetes/kubernetes/issues/123775).
+  The proposed approach is outlined in [here](#terminating-pods-and-terminal-job-conditions).
 - verify the validation passes during e2e tests for open-source projects (like Kueue and JobSet)
 - The feature flag enabled by default
 
+Second Beta (1.32):
+- Add validation rule that `Failed` and `Complete` conditions are added when
+  `terminating=0`, and `ready=0`.
```

Comment:
Either second beta, where we'd turn this on, first beta would be off by default to ensure the validation is in place. Or second alpha, and beta only in 1.31. See my other comment about it.

Reply:
As in the other comment, I think a second alpha would me more intuitive to users. Updated.